README.md

```diff
@@ -367,8 +366,8 @@ deallocation is performed (including in the destructor). The user must therefore
 the device active during _creation_ of an `rmm::device_buffer` matches the active device of the
 memory resource being used.
 
-Here is an incorrect example that creates a memory resource on device zero and then uses it to
-allocate a `device_buffer` on device one:
+Here is an incorrect example that creates a memory resource on device 0 and then uses it to
+allocate a `device_buffer` on device 1:
```

Comment:
Please no

Reply:
Device indices are numbers, not words. `CUDA_VISIBLE_DEVICES=zero` doesn't do the right thing, and you can't use a string when setting the current device. Can you provide a little more context for why words are your preference here? There are some grammar rules around spelling out numbers less than one hundred, but I don't think that is appropriate in this technical context.

Reply:
I would take ``` `0` ``` and ``` `1` ``` I suppose.

The code immediately below uses the numbers (rather than spelled out), so I don't think this is likely to cause confusion by spelling them out in the prose.